lib/bundle/mac_app_store_installer.rb

```diff
@@ -0,0 +1,21 @@
+module Bundle
+  class MacAppStoreInstaller
+    def self.install(name, id)
```

Comment:
Fail to check whether mas has been installed here. Also should we auto install it?